src/k8s/go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/canonical/k8s
 go 1.22.5
 
 require (
-	github.com/canonical/go-dqlite v1.21.0
-	github.com/canonical/lxd v0.0.0-20240709081608-9df996e36bac
-	github.com/canonical/microcluster v0.0.0-20240725000112-da36ecbd845e
+	github.com/canonical/go-dqlite v1.22.0
+	github.com/canonical/lxd v0.0.0-20240730172021-8e39e5d4f55f
+	github.com/canonical/microcluster v0.0.0-20240731145923-49573627997a
```

Comment:
this should go away, make sure to use `/v2` everywhere

Reply:
e.g. in places like https://github.com/canonical/k8s-snap/blob/032a5836d01450b7db612f83ddcb0fb9bf34c4f0/src/k8s/pkg/k8sd/database/feature_status.go#L11

You could use `| grep github.com/canonical/microcluster` to find instances where the `/v2` is not added in the imports